services/blockchain-indexer/shared/indexer/transactionProcessor/pos/stake.js

```diff
@@ -90,29 +93,51 @@ const decrementStakeTrx = async (stake, trx) => {
 	await stakesTable.decrement(decrementParam, trx);
 };
 
+const updateTotalStake = async (changeAmount, dbTrx) => {
```

Comment:
As discussed, please also add the logic for `totalSelfStake`.

Reply:
yeah, on it.